### ISSUE TITLE
Remove modal when creating a card

### ISSUE THREAD

**Issue by jrmneveu** [asdine/pulpe#57]
Display the input in the compact card.

**Comment by jrmneveu** [comment]
In this issue, css is no longer needed.

When the user click on "Add a new card" : 
- It shows the card in the edit mode, with the input and the focus.
- When the user press enter or focus out, it validates the card creation.

It would be interesting to think about creating a new card just after the previous one is validated. Putting the user immediately in a new card in edit mode with the focus. New issue ?

**Comment by asdine** [comment]
@jrmneveu Good idea! Yeah it's totally a new issue that will addressed on future releases
